dcc-submission-validator/src/test/java/org/icgc/dcc/submission/validation/accession/ega/EGAFileAccessionValidatorFtpProvider.java

```diff
@@ -0,0 +1,83 @@
+package org.icgc.dcc.submission.validation.accession.ega;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.ftpserver.FtpServer;
+import org.apache.ftpserver.FtpServerFactory;
+import org.apache.ftpserver.ftplet.FtpException;
+import org.apache.ftpserver.listener.ListenerFactory;
+import org.apache.ftpserver.usermanager.PropertiesUserManagerFactory;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import java.io.*;
+
+/**
+ * Copyright (c) 2017 The Ontario Institute for Cancer Research. All rights reserved.
+ * <p>
+ * This program and the accompanying materials are made available under the terms of the GNU Public License v3.0.
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ * <p>
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+public class EGAFileAccessionValidatorFtpProvider {
+  private static FtpServer ftpServer = null;
+  private static String adminHomePath = "/tmp/submission/ega/test";
+  private static File adminHomeDir = new File(adminHomePath);
+  private static String dataPath = adminHomePath + "/ICGC_metadata";
+  public static int defaultFtpPort = 2222;
+  @BeforeClass
+  public static void initialize() {
+
+    try {
+      if(adminHomeDir.exists())
+        FileUtils.deleteDirectory(adminHomeDir);
+
+      adminHomeDir.mkdirs();
+      (new File(dataPath)).mkdir();
+
+      FileOperationHelper.copyFileFromClasspathToTmpDir("/fixtures/validation/accession/ftp", "users.properties", adminHomePath);
+      FileOperationHelper.copyFileFromClasspathToTmpDir("/fixtures/validation/accession/ftp", "EGAD00001000045.tar.gz", dataPath);
```

Comment:
Actually I just saw that this is in the `test` folder hierarchy. If this is used for unit tests can you add `Test` to the classname so that is obvious at first read. Thanks.